UITests/run.py

```diff
@@ -61,8 +85,37 @@
 destination_arg = '-destination'
 
 for device in devices:
+	# get the simulator's uuid
+	list = subprocess.Popen('xcrun simctl list devices 13.5'.split(), stdout=subprocess.PIPE)
+	device_grep = subprocess.Popen(['grep', '{} ('.format(device)], stdin=list.stdout, stdout=subprocess.PIPE)
+	list.stdout.close()
+	uuid_grep = subprocess.Popen('grep -E -o -i ([0-9a-f]{8}-([0-9a-f]{4}-){3}[0-9a-f]{12})'.split(), stdin=device_grep.stdout, stdout=subprocess.PIPE)
+	device_grep.stdout.close()
+	simulator_uuid = uuid_grep.communicate()[0].decode().strip()
+
+	simulator_fontSize_preferences_file = '{}/Library/Developer/CoreSimulator/Devices/{}/data/Library/Preferences/com.apple.UIKit.plist'.format(os.path.expanduser("~"), simulator_uuid)
+	if os.path.exists(simulator_fontSize_preferences_file):
+		# shutdown the simulator in case it is running
+		print("Shutdown the {} simulator in case it is running".format(device))
+		subprocess.call(["xcrun", "simctl", "shutdown", device], stderr=open(os.devnull))
+
+		# set the simulator's font size
+		print("Set the {} simulator\'s font size to {}".format(device, font_size))
+		simulator_largerText_preferences_file = '{}/Library/Developer/CoreSimulator/Devices/{}/data/Library/Preferences/com.apple.preferences-framework.plist'.format(os.path.expanduser("~"), simulator_uuid)
```

Comment:
This file is not present until we manually enable the ```Larger Accessibility Sizes``` setting.
Anyway, this will be edited if present or created if not.